back/engines/commercial/verification/app/services/verification/verification_service.rb

```diff
@@ -112,10 +112,9 @@ def locked_custom_fields(user)
       custom_fields.uniq
     end
 
-    # Return meta data for use in permission actions
-    def action_metadata(method: nil)
-      method ||= first_method_enabled_for_verified_actions
-      return { allowed: false } unless method.respond_to?(:enabled_for_verified_actions?) && method&.enabled_for_verified_actions?
+    # Return method metadata
```

Comment:
Main change is to change this from 'metadata specific for verified actions' to just 'method metadata'.

One of the bugs was that I forgot that we only generate this metadata for verified action-supporting methods. I was assuming it would work for all methods, and I was using it to visualize locked fields and whatnot in the FE for those other methods. But I only tested it with the Fake SSO, which of course if also a verified-action supporting method, so I hadn't seen that it actually breaks the whole interface for other verification methods.